app/controllers/venues_controller.rb

```diff
@@ -30,6 +30,26 @@ def create
     end
   end
 
+
+  def show
+    # @booking = @venue.bookings.find { |booking| booking.user == current_user }
```

Comment:
@alexagodzilla, Do we need this line in?

Reply:
I mean does it need to be uncommented?

Reply:
Yes, we could remove that I think